Scripts/build_librustzcash.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+BASEPATH="${PWD}"
+TARGET_DIR="target"
+
+
+LIB_PATH="ZcashLightClientKit/$FLAVOR_FOLDER/zcashlc"
```

Comment:
If rust library takes more than 10 seconds to build then look into commiting compiled library and rewriting script to move it around